packages/next-plugin-sentry/config.js

```diff
@@ -0,0 +1,6 @@
+/**
+ * TODO(kamil): Unify SDK configuration options.
+ * Workaround for callbacks, integrations and any unserializable config options.
+ **/
+exports.serverConfig = {}
```

Comment:
The runtime configuration approach is not needed if you compile the configuration values into the bundle (prefixing using `NEXT_PUBLIC_`. However keep in mind that we shouldn't do that for the `getRelease` part given it will cause every single build to invalidate the webpack cache.

Reply:
Yes, but there's no way to provide additional configuration to `Sentry.init` without that approach, as `NEXT_PUBLIC_` supports only serializable values.